src/collections/Table/TableHeaderCell.js

```diff
@@ -8,23 +9,24 @@ import {
 } from '../../lib'
 import TableCell from './TableCell'
 
+/**
+ * A table can have a header cell.
+ */
 function TableHeaderCell(props) {
   const { as, className, sorted } = props
   const classes = cx(
     useValueAndKey(sorted, 'sorted'),
     className
   )
   const rest = getUnhandledProps(TableHeaderCell, props)
-  return <TableCell as={as} {...rest} className={classes} />
+
+  return <TableCell {...rest} as={as} className={classes} />
```

Comment:
Usually, when spreading this is true.   But, since `rest` excludes `as` and `className` due to `getUnhandledProps` there is no danger of overwriting.  I still think it is good to move it to the first position though, 👍